### ISSUE TITLE
a bug?

### ISSUE THREAD

**Issue by neouyghur** [carpedm20/ENAS-pytorch#49]
Hi, thanks for sharing the code. I am implementing the CNN part. I think the` block_idx` in `forward` function should be moded by 2 when CNN case as you used only two softmax. Could you check it? Thanks.

https://github.com/carpedm20/ENAS-pytorch/blob/25c4a89e17851d72e85566e16c81f0fd3749d58f/models/controller.py#L173-L177
